packages/plugins/EVM/src/state/IdentityService.ts

```diff
@@ -161,7 +161,7 @@ export class IdentityService extends IdentityServiceState<ChainId> {
         )
 
         if (!response.ok || !response.val.ownerAddress) return
-        return this.createSocialAddress(SocialAddressType.NEXT_ID, response.val.ownerAddress)
+        return this.createSocialAddress(SocialAddressType.Mask, response.val.ownerAddress)
```

Comment:
If somewhere used `SocialAddressType.NEXT_ID`, we must investigate whether or not to replace it with `SocialAddressType.Mask`.